types/telegram-web-app/index.d.ts

```diff
@@ -307,260 +439,202 @@ export interface WebApp {
-     * callback function will be called when the popup is closed and the first
-     * argument will be a boolean indicating whether the user granted this
-     * access.
+     * @since Bot API 6.2+
      */
-    requestWriteAccess(callback?: (success: boolean) => void): void;
+    showPopup(params: PopupParams, callback?: (button_id: string) => void): void;
+
     /**
-     * A method that shows a native popup prompting the user for their phone
-     * number.
+     * A method that shows message in a simple confirmation window with 'OK' and 'Cancel' buttons.
+     * If an optional callback parameter was passed, the callback function will be called when the popup is closed and the first argument will be a boolean indicating whether the user pressed the 'OK' button.
      *
-     * @param callback If an optional callback parameter was passed, the
-     * callback function will be called when the popup is closed and the first
-     * argument will be a boolean indicating whether the user shared its phone
-     * number. The second argument, contingent upon success, will be an object
-     * detailing the shared contact information or a cancellation response.
+     * @since Bot API 6.2+
      */
-    requestContact(callback?: (success: boolean, response: RequestContactResponse) => void): void;
-    /**
-     * A method that informs the Telegram app that the Web App is ready to be
-     * displayed. It is recommended to call this method as early as possible, as
-     * soon as all essential interface elements are loaded. Once this method is
-     * called, the loading placeholder is hidden and the Web App is shown. If
-     * the method is not called, the placeholder will be hidden only when the
-     * page is fully loaded.
-     */
-    ready(): void;
-    /**
-     * A method that expands the Web App to the maximum available height. To
-     * find out if the Web App is expanded to the maximum height, refer to the
-     * value of the Telegram.WebApp.isExpanded parameter
-     */
-    expand(): void;
-    /** A method that closes the Web App. */
-    close(): void;
+    showAlert(message: string, callback?: () => void): void;
+
     /**
-     * `True`, if vertical swipes to close or minimize the Mini App are enabled.
-     * `False`, if vertical swipes to close or minimize the Mini App are
-     * disabled. In any case, the user will still be able to minimize and close
-     * the Mini App by swiping the Mini App's header.
+     * A method that shows message in a simple confirmation window with 'OK' and 'Cancel' buttons.
+     * If an optional callback parameter was passed, the callback function will be called when the popup is closed and the first argument will be a boolean indicating whether the user pressed the 'OK' button.
+     *
+     * @since Bot API 6.2+
      */
-    isVerticalSwipesEnabled: boolean;
+    showConfirm(message: string, callback?: (success: boolean) => void): void;
```

Comment:
The docs say that the boolean indicates whether OK was pressed. The parameter was called `ok`. Why renamed it to success? Doesnt' that make it harder to understand?
```suggestion
    showConfirm(message: string, callback?: (ok: boolean) => void): void;
```

Reply:
Idea was to use standardized success callback for all methods

"ok callback" too specific